dts/bindings/fuel-gauge/battery_cutoff.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+
+  Properties for fuel-gauges that may control battery cutoff, this is common in SBS compliant or
+  similarly smart battery fuel gauges.
```

Comment:
Do we need the word 'battery' ?

Reply:
Would prefer to differentiate from other future forms of `cutoff` such as with charging.

Alternatively, I could rename all of this to ship_mode. I prefer the battery cutoff nomenclature because it's more self-defining.